public/data/Piyush903.json

```
@@ -0,0 +1,31 @@
+{
+    "name": "Piyush Nijhawan",
+    "location": "New Delhi,India",
+    "bio": "Full Stack Developer proficient in PHP,React",
+    "avatar": "https://github.com",
```

```suggestion
    "avatar": "https://github.com/Piyush903.png",
```

Add your username so avatar can be fetched.